### PR TITLE
Dbutson/dashboards

### PR DESCRIPTION
**Pull request by butson**
another set of updates to dashboards,  primarily adding descriptions,   variables selection based upon time window.

### CODE REVIEW

**Comment by butson** [comment]
If you start up docker container from this branch,  and point influxurl to http://monitor.don.mynuodb.com:8161 you'll have some data to look at changes.   You might need to zoom around to get a good time window for having data.

File: conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json [hunk 1]
```diff
@@ -194,6 +200,12 @@
       ],
       "datasource": null,
```
Comment (acabrele) [inline conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json]: I think there are a few examples of datasource: null instead of datasource: nuodb

**Review by acabrele — APPROVED** [review]
Had a poke around, I see fix to Aggregate Transaction Rate on Overview is fixed saving me the PR, thanks!

**Review by mkysel — APPROVED** [review]
looks good

**Pull request merged**